OCR/info-server/index.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const { exec } = require("child_process");
+
+const app = express();
+app.use(express.json());
+
+app.get("/info", (request, response) => {
+  try {
+    exec(
+      'kubectl get --raw "/apis/metrics.k8s.io/v1beta1/nodes"',
```

Comment:
was metrics server preinstalled on gcp? I tested it locally on Kind and had to run the commands below, maybe add this to README (or also whatever you configured):

```
To install Metric Server run:

`kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml`

If running on Kind or other unsecured environment also run:

`kubectl patch -n kube-system deployment metrics-server --type=json -p '[{"op":"add","path":"/spec/template/spec/containers/0/args/-","value":"--kubelet-insecure-tls"}]'`
```

Also CPU and maybe memory info could be useful so returning parsed data from this command would do
![image](https://github.com/Akatroj/Federated-Learning-in-MCC/assets/69090648/e399be2a-ece0-453d-994c-8518bf66a839)

Reply:
Metric server is not required to get nodes number. I dont install it in ec2 end everything works properly. 

In my opinion we can start training with only nodes number and if it will be not enought I will add aditional stats.